### PR TITLE
Dev.to scraper

### LINKED ISSUE #755: Dev.to Scrapper
# Details

Come up with a script that scraps `n` number of articles from `any category` such as Python, JS, Android provided by the user and stores all the `n` articles separately in PDFs. The PDF file should contain the title, article content, and author-name

User Input: `n` and `category`
Expected Output: n PDF files

### PR DESCRIPTION
**Pull request by Ayushjain2205**
# Description

Script to scrape articles from dev.to as per user's choice and save them as pdf

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Dev.to Scraper/scraper.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+from fpdf import FPDF
+  
+# Get input for category and number of articles
+category = input("Enter category: ")
+number_articles = int(input("Enter number of articles: "))
+
+url = 'https://dev.to/search?q={}'.format(category)
+
+# initiating the webdriver. Parameter includes the path of the webdriver.
+driver = webdriver.Chrome('C:\Webdrivers\chromedriver')
```
Comment (kaustubhgupta) [inline Dev.to Scraper/scraper.py:15]: Take the chrome driver path input from user

File: Dev.to Scraper/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Running this Script would allow the user to scrape any number of articles from [dev.to](https://dev.to/) from any category as per the user's choice
+
+## Setup instructions
+In order to run this script, you need to have Python and pip installed on your system. After you're done installing Python and pip, run the following command from your terminal to install the requirements from the same folder (directory) of the project.
+```
+pip install -r requirements.txt
+```
+As this script uses selenium, you will need to install the chrome webdriver from [this link](https://sites.google.com/a/chromium.org/chromedriver/downloads)
+
+Once the installation is complete, you need to add the webdriver in the path in the following line:
```
Comment (kaustubhgupta) [inline Dev.to Scraper/README.md:11]: Remove these lines (11 - 14) as input will be taken for this

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]
@Ayushjain2205 Good job 💯 💯

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by santushtisharma10 — APPROVED** [review]

**Comment by santushtisharma10** [comment]
@antrikshmisri

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..

**Pull request merged**